src/actions/content.js

```diff
@@ -10,22 +10,26 @@ import { MESSAGE_ERROR } from '../constants/messages';
 import { setMessage } from './application';
 
 export const SCHEMA_REQUESTED = 'SCHEMA_REQUESTED';
-export const requestSchema = () => ({
+export const requestSchema = ({ entityTypeId, bundle }) => ({
   type: SCHEMA_REQUESTED,
-  payload: {},
+  payload: { entityTypeId, bundle },
 });
 
 export const SCHEMA_LOADED = 'SCHEMA_LOADED';
-function* loadSchema() {
+function* loadSchema(action) {
+  const { entityTypeId, bundle } = action.payload;
```

Comment:
I thought about whether we want to cache schema for now, but I think till we are aware of all the potential complexities around that, not caching is the most sane approach.

Reply:
Yes. Ok. So I think this is good?

Reply:
Yes!